src/components/item/publish/CCLicenseSelection.tsx

```diff
@@ -51,60 +87,165 @@ const CCLicenseSelection: FC<Props> = ({ item, disabled }) => {
 
   useEffect(() => {
     if (settings?.ccLicenseAdaption) {
-      setOptionValue(settings.ccLicenseAdaption as CCLicenseAdaption);
+      // Handles old license formats.
+      if (['alike', 'allow'].includes(settings?.ccLicenseAdaption as string)) {
```

Comment:
Does the string union not type-check into `includes`?

Reply:
The issue here is that `cLicenseAdaption` is not typed into `item.settings` so it is `Anything` ...